src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -97,8 +103,13 @@ public Person toModelType() throws IllegalValueException {
             throw new IllegalValueException(Address.MESSAGE_CONSTRAINTS);
         }
         final Address modelAddress = new Address(address);
-        //There is no invalid additionalNotes since it can be empty or non-empty.
-        final AdditionalNotes modelAdditionalNotes = new AdditionalNotes(additionalNotes);
+
+        final AdditionalNotes modelAdditionalNotes;
```

Comment:
To align with the sequence of the fields in UI, the section of code on `AdditionalNotes` should be placed after the `moneyPaid` instead.